src/malva/java/math/BigDecimalTest.java

```diff
@@ -0,0 +1,24 @@
+package malva.java.math;
+
+import java.math.BigDecimal;
+
+import malva.TestCase;
+
+public class BigDecimalTest extends TestCase {
+  public static void testStrippingZeros() {
```

Comment:
Let's following naming conventions and call this `test<NameOfMethodBeingTested>` (see other tests)

Reply:
Ok